### PR TITLE
Added create project option in my circuits

### PR DESCRIPTION
**Pull request by yash15601**
Fixes https://github.com/CircuitVerse/CircuitVerse/issues/710

I have added create-project option in my circuits you can see screenshot below
### Screenshots of the changes (If any) -
Screenshot (42)

Moved PR from [here](https://github.com/CircuitVerse/CircuitVerse/pull/859)

### CODE REVIEW

**Comment by MrBartusek** [comment]
change
```
Fixes
#710
```
to
`Fixes #710`

**Comment by yash15601** [comment]
> change
> 
> ```
> Fixes
> #710
> ```
> 
> to
> `Fixes #710`

done

**Comment by yash15601** [comment]
@tachyons Review

File: app/views/users/logix/index.html.erb [hunk 1]
```diff
@@ -28,16 +28,14 @@
           </div>
       </div>
   <% end %>
-
   <div class ="row">
-    <h2 class="card-title"><%= @user.name %> - Projects | <a href=  "<%= profile_path(@user) %>"> View Profile </a> | <a href=  "<%= user_favourites_path(@user) %>"> View Favourites </a> </h2>
+    <h2 class="card-title"><%= @user.name %> - Projects | <a href= "<%= profile_path(@user) %>" target="_blank"> View Profile </a> | <a href= "<%= user_favourites_path(@user) %>" target="_blank"> View Favourites</a>| <a href= "/simulator" target="_blank">Create Project</a></h2>
```
Comment (tachyons) [inline app/views/users/logix/index.html.erb]: Why new page ?

**Comment by yash15601** [comment]
Done @tachyons  Review

**Comment by yash15601** [comment]
@tachyons  please review

**Comment by yash15601** [comment]
@nitin10s  Review Please

**Comment by yash15601** [comment]
@nitin10s  It seems like your right but making UI more userfriendly i thought we should include that create project option

**Comment by nitin10s** [comment]
I understand your concern, but the thing is - both the options are on the same screen, therefore making the "Create Project" option redundant. Moreover, we've removed the View Profile button as well and changed the View Favourites option. [You can refer to the prototype]

**Comment by yash15601** [comment]
Okay @nitin10s  thanks

**Comment by nitin10s** [comment]
@tachyons You may close this.

**Pull request closed without merging**